parts/footer.html

```diff
@@ -0,0 +1,97 @@
+    <div class="wp-block-column is-vertically-aligned-top" style="flex-basis:55%"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical"}} -->
+    <div class="wp-block-group"><!-- wp:image {"id":14,"width":"349px","height":"auto","sizeSlug":"large","linkDestination":"none","className":"size-full is-resized"} -->
+    <figure class="wp-block-image size-large is-resized size-full"><img src="" alt="" class="wp-image-14" style="width:349px;height:auto"/></figure>
+    <!-- /wp:image -->
+    
+    <!-- wp:paragraph {"align":"left","style":{"elements":{"link":{"color":{"text":"var:preset|color|font dark"}}},"typography":{"lineHeight":"1.7"},"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"0"}}},"textColor":"font dark"} -->
+    <p class="has-text-align-left has-font-dark-color has-text-color has-link-color" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:0;line-height:1.7">Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit<br></p>
+    <!-- /wp:paragraph --></div>
+    <!-- /wp:group --></div>
+    <!-- /wp:column -->
+    
+    <!-- wp:column {"verticalAlignment":"top","width":"45%","style":{"spacing":{"blockGap":"0","padding":{"right":"0","left":"0"}}}} -->
+    <div class="wp-block-column is-vertically-aligned-top" style="padding-right:0;padding-left:0;flex-basis:45%"><!-- wp:group {"className":"dswp-align-right-desktop-left-mobile","style":{"spacing":{"blockGap":"var:preset|spacing|80","margin":{"top":"0","bottom":"0"},"padding":{"right":"0","left":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","verticalAlignment":"top"}} -->
+    <div class="wp-block-group dswp-align-right-desktop-left-mobile" style="margin-top:0;margin-bottom:0;padding-right:0;padding-left:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:heading {"level":6,"style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|30"}}}} -->
+    <h6 class="wp-block-heading" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--30)">Placeholder</h6>
+    <!-- /wp:heading -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="#placeholder" data-type="link" data-id="https://www2.gov.bc.ca/">Placeholder</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="#placeholder" data-type="link" data-id="https://www2.gov.bc.ca/">Placeholder</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="#placeholder" data-type="link" data-id="https://www2.gov.bc.ca/">Placeholder</a></p>
```

Comment:
not sure why these are placeholders, usually the footer has the generic gov links, and don't need to be replaced, maybe the occasional one would have to but in general they don't